src/vue-components/morph.vue

```diff
@@ -1,98 +1,19 @@
-                  <inflectionattribute :data="nextGroup.groupingKey" :type="types.voice" :linkedfeatures="linkedfeatures" :grouplevel="3" @sendfeature="sendFeature"/>
-                </span>
-                <span class="alpheios-morph__groupseparator" v-if="group.groupingKey.isCaseInflectionSet && (group.groupingKey[types.number] || group.groupingKey[types.tense] || nextGroup.groupingKey[types.tense] || nextgroup.groupingKey[types.voice])">:</span>
-                <div v-for="infl in nextGroup.inflections"
-                  :class="groupClass(group)">
-                    <inflectionattribute :linkedfeatures="linkedfeatures" :type="types.grmCase" :grouplevel="4" :data="infl.groupingKey" @sendfeature="sendFeature"/>
-                    <inflectionattribute :linkedfeatures="linkedfeatures" :type="types.gender" :grouplevel="4" :data="infl.groupingKey" :decorators="['parenthesize','abbreviate']"
-                      @sendfeature="sendFeature" v-if="! featureMatch(infl.groupingKey[types.gender],lex.lemma.features[types.gender])" />
-                    <inflectionattribute :linkedfeatures="linkedfeatures" :type="types.comparison" :grouplevel="4" :data="infl.groupingKey" @sendfeature="sendFeature"/>
-                    <inflectionattribute :data="infl.groupingKey" :type="types.person" :linkedfeatures="linkedfeatures" :grouplevel="4" :decorators="['appendtype']" @sendfeature="sendFeature"/>
-                    <inflectionattribute :data="infl.groupingKey" :type="types.number" :linkedfeatures="linkedfeatures" :grouplevel="4" @sendfeature="sendFeature"
-                      v-if="! group.groupingKey.isCaseInflectionSet"/>
-                    <inflectionattribute :data="infl.groupingKey" :type="types.tense" :linkedfeatures="linkedfeatures" :grouplevel="4" @sendfeature="sendFeature"
-                      v-if="! group.groupingKey.isCaseInflectionSet"/>
-                    <inflectionattribute :data="infl.groupingKey" :type="types.mood" :linkedfeatures="linkedfeatures" :grouplevel="4" @sendfeature="sendFeature"
-                      v-if="! group.groupingKey.isCaseInflectionSet"/>
-                    <inflectionattribute :data="infl.groupingKey" :type="types.voice" :linkedfeatures="linkedfeatures" :grouplevel="4" @sendfeature="sendFeature"
-                      v-if="! group.groupingKey.isCaseInflectionSet"/>
-                    <span v-for="item in infl.inflections">
-                      <inflectionattribute :data="item" type="example" :linkedfeatures="linkedfeatures" @sendfeature="sendFeature"/>
-                    </span>
-                </div><!-- end infl -->
-              </div><!-- end forms -->
-            </div><!-- end groupinflections -->
-          </div>
-        </div>
-      </div><!-- end alpheios-morph__inflections -->
-    </div>
-  </div>
+  <div>
+    <morph-inner
```

Comment:
At the very start of the working on this issue there were thoughts about having an ability to choose a layout based on user's preference. So I have organized different layouts using switching between different variants of morph-inner.
So it it would be decided to work with one variant - it would be easily integrated.
If it would be decided to work with different variants at once (using some switch) it could be done too.

Reply:
yes, let's keep the separate components in place. We can worry about passing the name of the one to use to the parent component later if and when that feature is actually requested :-)